dll/cpl/input/layout_list.c

```diff
@@ -87,51 +87,8 @@ LayoutList_Destroy(VOID)
 
 typedef HRESULT (WINAPI *FN_SHLoadRegUIStringW)(HKEY, LPCWSTR, LPWSTR, DWORD);
 
-/* FIXME: Use shlwapi!SHLoadRegUIStringW instead when it is fully implemented */
```

Comment:
Note that this is also in https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/kbswitch/kbswitch.c;hb=4cdcea5e65326cbea77458a3e9daa335d2e530b7#l121

Reply:
https://github.com/reactos/reactos/pull/5260